src/bootstrap/build/sanity.rs

```diff
@@ -78,6 +79,24 @@ pub fn check(build: &mut Build) {
             panic!("the iOS target is only supported on OSX");
         }
 
+        if cfg!(windows) {
```

Comment:
Right, but that's not a Windows specific limitation, right? If rustc ever dynamically links to LLVM (on any platform) we need the library directory to be in the dynamic library path?

It looks like the difference is just that the dynamic library directory on Windows happens to be the same as the binaries directory?